tests/test_modeling_utils.py

```diff
@@ -34,6 +34,8 @@
 from transformers import (
     AutoConfig,
     AutoModel,
+    AutoModelForSequenceClassification,
```

Comment:
```suggestion
    AutoModelForCausalLM,
```

Reply:
Testing your suggestions – this one seems necessary. The error arises when loading a model with missing keys. In our case it's loading a vanilla base model in SEQ_CLS mode, plus a LoRA regression head trained for SEQ_CLS.

Reply:
Awesome ok - sounds good then, could you add a comment when you load the model explaining this?

Reply:
Done.

Reply:
Thanks so much!